config/tools.json

```diff
@@ -7,15 +7,29 @@
         "description": "Explore APIs and companies with public AsyncAPI specifications.",
         "links": {
           "websiteUrl": "https://apitracker.io/specifications/asyncapi",
-          "repoUrl": ""
+          "repoUrl": "https://github.com/apideck-io"
         },
         "filters": {
-          "language": [],
-          "technology": [],
-          "categories": [
-            "api",
-            "directory"
+          "language": [
+            {
+              "name": "JavaScript",
+              "color": "bg-[#F2F1C7]",
+              "borderColor": "border-[#BFBE86]"
+            }
           ],
+          "technology": [
+            {
+              "name": "Node.js",
+              "color": "bg-[#BDFF67]",
+              "borderColor": "border-[#84CE24]"
+            },
+            {
+              "name": "TypeScript",
+              "color": "bg-[#61d0f2]",
+              "borderColor": "border-[#40ccf7]"
+            }
+          ],
+          "categories": ["api", "directory"],
```

Comment:
Please revert all the changes you've made in the categories arrays, this refactoring wasn't necessary.

Furthermore, the original formatting of the categories is beneficial as it makes adding new items to arrays much easier.